src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file. I see why it's beneficial to have when running tests locally, but when running the entire test suite in CI it's flooding the build logs.